### DISCUSSION TITLE
migration between clusters incl. pvc?

### DISCUSSION THREAD

**Discussion by KlavsKlavsen** [liqotech/liqo#1180] [General]
How do you handle PVC objects (and PVs) currently? if one was to use this on clusters at 2 providers (GKE and AWS f.ex.) - pods which need a PVC cannot just be restarted on "the other cluster" - as the pvc will not be available there, by defaulut atleast :)

**Comment by aleoli** [comment]
Hi @KlavsKlavsen! Liqo defines its own "virtual" storage class (`liqo`). When a PVC is created with this storage class you will have 2 different behaviors in the case of the pod is scheduled locally or not:
* __local pods__: a copy "real" PVC is created in the `liqo-storage` namespace using the default storage class (or the one you set during the liqo installation), then the PV spec is copied inside the virtual PV that your pods will mount. The liqo PV will have as node selector the one provided by the storage provider with the addition of the condition to be in the remote cluster
* __remote pods__: a "real" PVC is created in the remote cluster in the offloaded namespace using the default storage class (or the one you set during the liqo installation), then it creates a local placeholder PV. The liqo PV will have as node selector the condition to be in the remote cluster

The node selectors force the pod to be rescheduled in the same cluster where the PVCs are. You can change the cluster where the PVCs are using `liqoctl`. With `liqoctl move` you are able to move an unbound volume by selecting a target node (then cluster).

It moves the volume by backupping it with Restic as described in the issue https://github.com/liqotech/liqo/issues/1071
